src/it/java/com/google/checkstyle/test/base/AbstractModuleTestSupport.java

```diff
@@ -301,15 +301,15 @@ protected static String getCheckMessage(Class<? extends AbstractViolationReporte
 
     /**
      * Gets the check message 'as is' from appropriate 'messages.properties' file.
-     * @param messages The map of messages to scan.
+     * @param moduleConfig The configuration to scan for messages.
      * @param messageKey the key of message in 'messages.properties' file.
      * @param arguments the arguments of message in 'messages.properties' file.
      * @return The message of the check with the arguments applied.
      */
-    protected static String getCheckMessage(Map<String, String> messages, String messageKey,
+    protected static String getCheckMessage(Configuration moduleConfig, String messageKey,
```

Comment:
what is reason of making code smaller ?
now it become not clear "getCheckMessage on checkConfig and msgPreceded" 
if put in arguemnt big object  and use only one item from it, good sign that smth is wrong is design.

you can optimize horizontal space by:
```
Map<String, String> messages = checkConfig.getMessages();
final String[] expected = {
            "12:16: " + getCheckMessage(messages, msgPreceded, "<"),
}
```

Reply:
Done.